stan/math/prim/prob/multi_normal_cholesky_lpdf.hpp

```diff
@@ -119,40 +122,175 @@ return_type_t<T_y, T_loc, T_covar> multi_normal_cholesky_lpdf(
     logp += NEG_LOG_SQRT_TWO_PI * size_y * size_vec;
   }
 
-  const matrix_partials_t inv_L_dbl
-      = mdivide_left_tri<Eigen::Lower>(value_of(L_ref));
-
   if (include_summand<propto, T_y, T_loc, T_covar_elem>::value) {
+    Eigen::Matrix<T_partials_return, Eigen::Dynamic, Eigen::Dynamic>
+        y_val_minus_mu_val(size_y, size_vec);
+
     for (size_t i = 0; i < size_vec; i++) {
       decltype(auto) y_val = as_value_column_vector_or_scalar(y_vec[i]);
       decltype(auto) mu_val = as_value_column_vector_or_scalar(mu_vec[i]);
+      y_val_minus_mu_val.col(i) = y_val - mu_val;
+    }
+
+    matrix_partials_t half, scaled_diff;
```

Comment:
We just tend not to do the multi define

```suggestion
    matrix_partials_t half;
    matrix_partials_t scaled_diff;
```

Reply:
I was feeling adventurous :P